projects/plugins/jetpack/extensions/shared/components/audio-player/index.jsx

```diff
@@ -51,6 +51,7 @@ function AudioPlayer( {
 	onMetadataLoaded,
 } ) {
 	const audioRef = useRef();
```

Comment:
I think we should get rid of this reference. The `containerRef` should be enough to deal with both DOM references.

Reply:
I was thinking that, but we need to get at the audio element a lot and it's changing, so it felt like we'd keep having to search to find it in the DOM.

Reply:
it's doable using the container reference:

```es6
const audioEl = containerRef?.current.querySelector( 'audio' );
```

Anyway, it's up to you.

Reply:
Yes, that's the thing, we'd need to do that in all the methods that interact with the audio element. Do you think it would be better to do this or cache it in a reference? I'm not sure!